phpunit tests are failing in CI

Build link: https://drone.owncloud.com/owncloud/encryption/3060/4/4

Failure:
```bash
There were 2 errors:

1) OCA\Encryption\Tests\Crypto\EncryptAllTest::testCreateKeyPairs
TypeError: Argument 1 passed to Mock_OutputFormatterInterface_70102a73::setDecorated() must be of the type bool, null given, called in /var/www/owncloud/server/lib/composer/symfony/console/Helper/Helper.php on line 176

/var/www/owncloud/server/lib/composer/symfony/console/Helper/Helper.php:176
/var/www/owncloud/server/lib/composer/symfony/console/Helper/ProgressBar.php:518
/var/www/owncloud/server/lib/composer/symfony/console/Helper/ProgressBar.php:578
/var/www/owncloud/server/lib/composer/symfony/console/Helper/ProgressBar.php:591
/var/www/owncloud/server/lib/composer/symfony/console/Helper/ProgressBar.php:415
/var/www/owncloud/server/lib/composer/symfony/console/Helper/ProgressBar.php:324
/var/www/owncloud/server/apps/encryption/lib/Crypto/EncryptAll.php:198
/var/www/owncloud/server/tests/lib/TestCase.php:189
/var/www/owncloud/server/apps/encryption/tests/unit/Crypto/EncryptAllTest.php:244
phpvfscomposer:///var/www/owncloud/server/lib/composer/phpunit/phpunit/phpunit:106

2) OCA\Encryption\Tests\Crypto\EncryptAllTest::testEncryptAllUsersFiles
TypeError: Argument 1 passed to Mock_OutputFormatterInterface_70102a73::setDecorated() must be of the type bool, null given, called in /var/www/owncloud/server/lib/composer/symfony/console/Helper/Helper.php on line 176

/var/www/owncloud/server/lib/composer/symfony/console/Helper/Helper.php:176
/var/www/owncloud/server/lib/composer/symfony/console/Helper/ProgressBar.php:518
/var/www/owncloud/server/lib/composer/symfony/console/Helper/ProgressBar.php:578
/var/www/owncloud/server/lib/composer/symfony/console/Helper/ProgressBar.php:591
/var/www/owncloud/server/lib/composer/symfony/console/Helper/ProgressBar.php:415
/var/www/owncloud/server/lib/composer/symfony/console/Helper/ProgressBar.php:324
/var/www/owncloud/server/apps/encryption/lib/Crypto/EncryptAll.php:233
/var/www/owncloud/server/tests/lib/TestCase.php:189
/var/www/owncloud/server/apps/encryption/tests/unit/Crypto/EncryptAllTest.php:289
phpvfscomposer:///var/www/owncloud/server/lib/composer/phpunit/phpunit/phpunit:106

ERRORS!
Tests: 261, Assertions: 740, Errors: 2.

Generating code coverage report in Clover XML format ... done [00:00.035]
make: *** [Makefile:91: test-php-unit-dbg] Error 2
```
Related core issue: https://github.com/owncloud/core/issues/39630

--- a/tests/unit/Crypto/EncryptAllTest.php
+++ b/tests/unit/Crypto/EncryptAllTest.php
@@ -28,6 +28,7 @@ use OC\Files\Storage\Common;
 use OC\Files\View;
 use OCA\Encryption\Crypto\EncryptAll;
 use OCP\Files\Mount\IMountPoint;
+use Symfony\Component\Console\Formatter\OutputFormatterInterface;
 use Symfony\Component\Console\Helper\ProgressBar;
 use Symfony\Component\Console\Output\NullOutput;
 use Symfony\Component\Console\Output\OutputInterface;
@@ -122,8 +123,11 @@ class EncryptAllTest extends TestCase {
 		$this->userInterface = $this->getMockBuilder(UserInterface::class)
 			->disableOriginalConstructor()->getMock();
 
+		$outputFormatterInterface = $this->createMock(OutputFormatterInterface::class);
+		$outputFormatterInterface->expects($this->any())->method('isDecorated')
+			->willReturn(false);
 		$this->outputInterface->method('getFormatter')
-			->willReturn($this->createMock('\Symfony\Component\Console\Formatter\OutputFormatterInterface'));
+			->willReturn($outputFormatterInterface);
 
 		$this->userManager->method('getBackends')->willReturn([$this->userInterface]);
 		$this->userInterface->method('getUsers')->willReturn(['user1', 'user2']);

